### DISCUSSION TITLE
raudio site on samsung tv shows chinese characters

### DISCUSSION THREAD

**Discussion by cYCL157** [rern/rAudio#239] [Fixed Issues]
![2021-02-12 08 18 34](https://user-images.githubusercontent.com/275680/110191289-c4a38b80-7de4-11eb-9e19-6e7317b30095.jpg)

I use my samsung tv sometimes to control raudio, and while it works great I get some Chinese characters for the buttons and icons.  I looked for settings thinking there was some locale setting that I could change, but couldn't find one.

Any suggestions??

**Comment by AndyFx12** [comment]
Hi cYCL157,
it looks like a similar "problem" on my Galaxy-Watch. I could fix them by loading /srv/http/assets/fonts/enhance.ttf to 
[https://fontforge.org/en-US/](url). 
Without any changes "generate font" (there are some errors found - ignore them) and overwrite enhance.ttf.
My settings on generating the font:
![grafik](https://user-images.githubusercontent.com/35779271/110232368-7645d800-7f1d-11eb-8634-228305598c19.png)

Be warned: Every update overwrites this modded font with original.

**Reply by rern** [reply]
@AndyFx12,

Thank you for the clue. Next update will have this fixed.

**Reply by AndyFx12** [reply]
@rern 
New fonts from update-branch work fine on my Galaxy-Watch Samsung-Internet-Browser.
Thanks!

**Reply by cYCL157** [reply]
Thank you both!!!  Updated to new version and it works great!

**Comment by cYCL157** [comment]
I just noticed that this issue came back -- I am running 20211114, but have not idea if it's been there for awhile or not, just happened to notice today when I pulled it up on my tv.
